pages/[...slug].vue

```diff
@@ -18,6 +18,15 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <div>
+        <nav>
+          <span v-for="x in breadcrumbs" :key="x._path" class="flex-row mx-0.5">
+            <NuxtLink :to="x._path"
+              ><button>{{ x.title }}{{ '/' }}</button></NuxtLink
+            >
```

Comment:
This formatting looks awful

```suggestion
            <NuxtLink :to="x._path">
              <button>{{ x.title }}{{ '/' }}</button>
            </NuxtLink>
```

Reply:
fixed